src/main/java/io/xpring/javascript/JavaScriptSigner.java

```diff
@@ -29,8 +29,12 @@ public JavaScriptSigner() throws JavaScriptLoaderException {
 
     /**
      * Sign the given transaction with the given wallet.
+     *
+     * @throws JavaScriptLoaderException An exception if the javascript could not be loaded.
```

Comment:
Minor nit, but generally this should come last in the Javadoc. For future, consider using checkstyle here -- see https://github.com/hyperledger/quilt for details.

Reply:
Linting is a tech debt cleanup task that I'm tracking in Asana :)  I've blocked a day a week as sheerly tech debt clean up days so this is a thing that will get burned down soon.

Reply:
Added throws